src/e2e/resources/data/InstructorCourseEditPageE2ETest.json

```diff
@@ -7,26 +7,56 @@
       "readNotifications": {}
     },
      "ICEdit.observer": {
-      "googleId": "tm.e2e.ICEdit.observer",
-      "name": "Teammates Instructor",
-      "email": "ICEdit.observer@gmail.tmt",
-      "readNotifications": {}
-    }
+       "googleId": "tm.e2e.ICEdit.observer.CS2104",
+       "name": "Teammates Instructor",
+       "email": "ICEdit.observer@gmail.tmt",
+       "readNotifications": {}
+     }
```

Comment:
I'm nitpicking here, but the wrong indentation is not the one you have fixed here; it's the one line up there `"ICEdit.observer": {`

Reply:
Good catch. It was actually done by IDE and I didn't even notice. Thanks.